src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -296,11 +298,9 @@ public void run() {
     }
 
     private void applySubsetMultiThread(Path tmpDir, GoGoDuckModule module, int threadCount) throws GoGoDuckException {
-        userLog.log(String.format("Applying subset '%s'", module.getNcksSubsetParameters().toString()));
         logger.info(String.format("Applying subset on directory '%s'", tmpDir));
 
         File[] directoryListing = tmpDir.toFile().listFiles();
-        logger.info(String.format("Subset for operation is '%s'", module.getNcksSubsetParameters()));
```

Comment:
We've lost some useful logging here with the removal of the loggin statements in this class which I've actually used before - should do it somewhere.